crates/proto/Cargo.toml

```diff
@@ -25,42 +25,45 @@ categories.workspace = true
 license.workspace = true
 
 [features]
-async-std = ["dep:async-std", "dep:pin-utils", "dep:socket2"]
+default = ["std", "tokio"]
```

Comment:
Please keep the features in the existing order (or reorder in a separate commit) to make these changes easier to review, please.

Reply:
This first commit still moves up `default`. Please keep it in the same place as before in the commit that introduces the `std` feature.